src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -85,14 +85,17 @@
 
     private final BidderCatalog bidderCatalog;
     private final BidderParamValidator bidderParamValidator;
+    private final String blacklistedApps;
```

Comment:
i'd like to present blacklisted apps as a list. That makes app validation more fair, cause now it is like substring checking. What if YML configured to `Black-Super-Listed-App` value but in incoming prebid request we obtain `Super-Listed-App`? ;-)
Pls see `gdpr.eea-countries` as an example.

Reply:
Thanks! I did check the `eea-countries`, but didn't check any further as I saw it getting passed through `@Value` as a String for the first time.